test/unit/deferred.js

```diff
@@ -710,13 +710,12 @@ QUnit.test( "jQuery.when", function( assert ) {
 
 QUnit.test( "jQuery.when - joined", function( assert ) {
 
-	assert.expect( 195 );
+	assert.expect( 81 );
```

Comment:
Yes we did. However, the docs say nothing about it, and it seems strange to pass through the notifications from the subordinates all mixed into the master deferred.